src/components/Layout/Platform/Tabs.tsx

```diff
@@ -0,0 +1,44 @@
+import Link from "next/link"
+import { useRouter } from "next/router"
+import React from "react"
+import { Nav, NavItem, NavLink } from "reactstrap"
+
+import { paths } from "Shared/constants"
+import { urlPartReplacer } from "Shared/urlParams"
+
+import { ErddapMoreInfoDropdown, ErddapObservedDropdown, ForecastDropdown } from "Features/ERDDAP"
+import { PlatformFeature } from "Features/ERDDAP/types"
+
+interface PlatformTabsProps {
+  platform: PlatformFeature
+}
+
+export const PlatformTabs: React.FC<PlatformTabsProps> = ({ platform }) => (
+  <Nav tabs={true}>
+    <ErddapObservedDropdown platform={platform} />
+    <Tab to={paths.platforms.platform} name="Latest Conditions" id={platform.id} />
```

Comment:
Codacy found an issue: ['Tab' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6288046)